### PR TITLE
Run rcmdcheck as cran in CI

### PR DESCRIPTION
**Pull request by keller-mark**
Towards https://github.com/keller-mark/pizzarr/issues/95

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -52,17 +61,26 @@ jobs:
...
+            error_on = "error",
             check_dir = "check"
           )
         shell: Rscript {0}
         env:
           _R_CHECK_FORCE_SUGGESTS_: false
+      - name: rcmdcheck as CRAN
+        run: |
+          rcmdcheck::rcmdcheck(
+            args = "--as-cran",
```
Comment (dblodgett-usgs) [inline .github/workflows/deploy.yml:73]: --no-manual to avoid the pdftatex errors. Not sure we need to build the full manual.

**Comment by dblodgett-usgs** [comment]
In case you want to pull the plug and just use the community actions, this passes three OSes but doesn't do the pkgdown build. https://github.com/dblodgett-usgs/pizzarr/blob/main/.github/workflows/deploy.yml

**Comment by keller-mark** [comment]
Ah yes that looks much simpler

**Comment by dblodgett-usgs** [comment]
Lemme see if I can get it to work the same as the build here. Will open a PR if I can figure it out.

**Pull request closed without merging**